week_02/src/databaseCopy.js

```diff
@@ -0,0 +1,7 @@
+import mongoose from "mongoose";
+
+const server = "mongodb2";
+const local = "localhost";
+const database = "cabify_bootcampCopy";
+
+export default mongoose.createConnection(`mongodb://${local}/${database}`, { useNewUrlParser: true });
```

Comment:
Try to have only one file (merge database and databaseCopy) and create only one function with a parameter that will be the dataaseName.